pandas/io/orc.py

```diff
@@ -52,3 +66,109 @@ def read_orc(
     with get_handle(path, "rb", is_text=False) as handles:
         orc_file = orc.ORCFile(handles.handle)
         return orc_file.read(columns=columns, **kwargs).to_pandas()
+
+
+def to_orc(
+    df: DataFrame,
+    path: FilePath | WriteBuffer[bytes] | None = None,
+    *,
+    engine: Literal["pyarrow"] = "pyarrow",
+    index: bool | None = None,
+    engine_kwargs: dict[str, Any] = {},
```

Comment:
Sorry just realized. Could you default this here `engine_kwargs: dict[str, Any] | None = None,` and if `None` convert to empty dict in the function?

Reply:
Done!